source/apache-way/apache-project-maturity-model.md

```diff
@@ -191,76 +186,129 @@ access or decision power, and applies these principles consistently.
 
 <dt id="CO60">CO60</dt>
 <dd>
-The community operates based on consensus of its members (see CS10) who 
-have decision power. Dictators, benevolent or not, are not welcome in 
-Apache projects. 
+The community operates based on consensus of its members (see CS10) who
+have decision power. Dictators, benevolent or not, are not welcome in
+Apache projects.
 </dd>
 
 <dt id="CO70">CO70</dt>
 <dd>
 The project strives to answer user questions in a timely manner.
 </dd>
+</dl>
+
+## Trademark and Branding
+
+Notable items from the [Trademark Policy][trademark-policy],
+[FAQ][trademark-faq], [Guide][trademark-guide], and
+[PMC Branding Responsibilities][branding-responsibilities].
+The project's PMC should comply with ASF's branding and trademark policy in
+all places it controls.
+
+[trademark-policy]: https://www.apache.org/foundation/marks/
+[trademark-faq]: https://www.apache.org/foundation/marks/faq/
+[trademark-guide]: https://www.apache.org/foundation/marks/guide
+[branding-responsibilities]: https://www.apache.org/foundation/marks/responsibility
```

Comment:
Per discussion it's possible we link only PMC Branding Responsibilities here and improve it as the portal to all the other materials. But that would be another discussion and I'd start a discussion at trademarks@ later to move it forward, and then come back here.